git_history_of_project/git_history_of_project.md

```diff
@@ -0,0 +1,503 @@
+<!--
+
+author:  Elizabeth Drellich
+email:    drelliche@chop.edu
+version:  1.0.0
+module_template_version: 2.0.0
+language: en
+narrator: UK English Female
+title: Exploring the History of your Git Repository
+comment:  This module will teach you how to look at past versions of your work on Git and compare your project with previous versions.
+long_description: You know that version control is important. You know how to save your work to your Git repository. Now you are ready to look at and compare different versions of your work. In this module you will you will learn how to navigate through the commits you have made to Git. You will also learn how to compare current code with past code.
+
+estimated_time: 30 minutes
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- Identify and use the `HEAD` of a repository.
+- Identify and use Git commit numbers.
+- Compare versions of tracked files.
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# Exploring the History of a Git Repository
+
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:**
+@estimated_time
+
+**Pre-requisites**
+
+This module is for you if:
+
+* You know know how to create a Git repository and make changes using `add` and `commit` from a command line interface (CLI).
+* You want to learn how to explore and compare different versions of your work.
+
+
+**Learning Objectives**
+
+@learning_objectives
+
+</div>
+
+## Lesson Preparation
+
+The purpose of this module is to learn how to explore the history of a Git repository that has already been created. We are going to download Dracula's repository in which he started documenting the pros and cons of moving with his friends to Mars.
+
+Open up your command line interface and navigate to a place where you want to put a new repository. This might be your `Desktop` on a Mac or `C` on a computer running Windows, but it can really be anywhere on your computer you choose. Once you are in the location you want to put Dracula's repository, type:
```

Comment:
Good point! But if they are in Git Bash like they need to be, the slashes will be `/` instead of `\`when they navigate to it. What do you think about just saying 'Desktop' for everyone?

```suggestion
Open up your command line interface and navigate to a place where you want to put a new repository. This might be your `Desktop` but it can really be anywhere on your computer you choose. Once you are in the location you want to put Dracula's repository, type:
```